### DISCUSSION TITLE
Confusion starting new project

### DISCUSSION THREAD

**Discussion by leon0399** [microsoft/react-native-windows#10939] [Q&A] [answered]
I'm planning to develop a new project with React Native Windows and I'm VERY confused about Windows-oriented terms

My plan is to develop software, using monorepo for iOS/Android/Windows

I'm considering using WinUI 3 for parts of the Windows app (title bar, nav pane), given the fact that the design was made with Win UI 3 Toolkit (you can see design references below)  
I know, that documentation claims, that WinUI support is experimental, and I'm fine using experimental features

<details>
  <summary>Design Examples</summary>

  ### Target design:
  ![Devices](https://user-images.githubusercontent.com/1759654/204372962-5d918df2-14f0-423a-8dbb-e70eb94781d1.png)

  ### Design inspiration:
  ![image](https://user-images.githubusercontent.com/1759654/204373211-4cf899b2-8244-48a9-bc95-99afa2e481b9.png)

</details>

But I was extremely confused when I started noticing the following terms in other places, such as:

- WinAppSDK
- Windows App SDK (same as above?)
- Project Reunion
- UWP

If I understood correctly, WinUI 3 is a graphical framework for WinAppSDK. But if WinAppSDK is already supported, how does the usage of WinUI 3 prevent me from using Native Modules (see below)?

<details>
  <summary>Excerpts from documentation</summary>

  ![image](https://user-images.githubusercontent.com/1759654/204374627-7185ce8f-a914-43e0-8f0e-168a12baeafa.png)
</details>

**What is the best approach in my case?** Use RN with WinUI? Use RN without UI? Develop an application with WinUI without RN?
My goal is to use native components, to achieve cool Fluent UI features, such as Acrylic Blur, Mica blur, etc...
It will also need to run multiple WebSocket servers, as well as connecting to Bluetooth/BLE devices, so I'm guessing native modules are really necessary 

Please, keep in mind, that I have a lot of experience in developing web React projects, but I'm a novice in developing native apps with both RN and C# projects

**Comment by chrisglein** [comment, marked as answer]
This article may help with introducing yourself to native Windows development via RN: https://microsoft.github.io/react-native-windows/docs/native-code

> WinAppSDK
Windows App SDK (same as above?)
Project Reunion

These are all the same thing. Project Reunion was the codename for what became the Windows App SDK. Which is sometimes shortened to WinAppSDK or WASDK. Hopefully there aren't any references to Reunion in our RN docs. If that's true, let us know so we can correct that. We definitely want to iron out any inconsistency to avoid confusion.

You can find more information about WinUI3 means relative to WinAppSDK here: http://aka.ms/winui3

If you are new to native development on Windows, and are coming through RN, I wouldn't recommend using the experimental WinUI3 support. None of the existing RN modules that support Windows will support WinUI3 yet. And your UI requirements should be served jsut fine by the default mode for RN, which is WinUI2.

**Reply by leon0399** [reply]
Thank you so much for the clarification 🔥
